src/components/PeopleManagement/DeleteGroupModal.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useParams } from 'react-router-dom';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { logError } from '@edx/frontend-platform/logging';
+import {
+  ActionRow, Button, ModalDialog, useToggle,
+} from '@openedx/paragon';
+import { RemoveCircleOutline } from '@openedx/paragon/icons';
+
+import GeneralErrorModal from './GeneralErrorModal';
+import { ROUTE_NAMES } from '../EnterpriseApp/data/constants';
+
+import LmsApiService from '../../data/services/LmsApiService';
+
+const DeleteGroupModal = ({
+  group, isOpen, close,
+}) => {
+  const { enterpriseSlug } = useParams();
+  const [isErrorOpen, openError, closeError] = useToggle(false);
+  const removeEnterpriseGroup = async () => {
+    try {
+      const response = await LmsApiService.removeEnterpriseGroup(group?.uuid);
+      if (response.status === 204) {
```

Comment:
we can remove this check since the promise is already wrapped in a `try...catch`